ui/index.html

```diff
@@ -19,7 +19,7 @@
     <body>
 
          <!-- HEADER -->
-        <div data-widget="topnavigation"><!-- --></div>
+        <div data-widget="topnavigation" data-autologin="true"><!-- --></div>
```

Comment:
Yea, I don't think it makes sense to have it if you land on a content page, for example. So it's only valid functionality for the signup page and "landing" page.